bqskit/passes/__init__.py

```diff
@@ -329,6 +330,7 @@
     'UpdateDataPass',
     'ToU3Pass',
     'ScanningGateRemovalPass',
+    'TreeScanningGateRemovalPass',
```

Comment:
Also add the to the documentation docstring in this file

Reply:
Does that just mean adding the TreeScanningGateRemovalPass where I did in my latest commit?